db/internal_stats.cc

```diff
@@ -989,32 +989,36 @@ void InternalStats::DumpCFMapStats(
 }
 
 void InternalStats::DumpCFMapStatsIOStalls(
-        std::map<std::string, std::string>* cf_stats) {
+    std::map<std::string, std::string>* cf_stats) {
   (*cf_stats)["io_stalls.level0_slowdown"] =
-          std::to_string(cf_stats_count_[LEVEL0_SLOWDOWN_TOTAL]);
+      std::to_string(cf_stats_count_[LEVEL0_SLOWDOWN_TOTAL]);
   (*cf_stats)["io_stalls.level0_slowdown_with_compaction"] =
-          std::to_string(cf_stats_count_[LEVEL0_SLOWDOWN_WITH_COMPACTION]);
+      std::to_string(cf_stats_count_[LEVEL0_SLOWDOWN_WITH_COMPACTION]);
   (*cf_stats)["io_stalls.level0_numfiles"] =
-          std::to_string(cf_stats_count_[LEVEL0_NUM_FILES_TOTAL]);
+      std::to_string(cf_stats_count_[LEVEL0_NUM_FILES_TOTAL]);
   (*cf_stats)["io_stalls.level0_numfiles_with_compaction"] =
-          std::to_string(cf_stats_count_[LEVEL0_NUM_FILES_WITH_COMPACTION]);
+      std::to_string(cf_stats_count_[LEVEL0_NUM_FILES_WITH_COMPACTION]);
   (*cf_stats)["io_stalls.stop_for_pending_compaction_bytes"] =
-          std::to_string(cf_stats_count_[HARD_PENDING_COMPACTION_BYTES_LIMIT]);
+      std::to_string(cf_stats_count_[HARD_PENDING_COMPACTION_BYTES_LIMIT]);
   (*cf_stats)["io_stalls.slowdown_for_pending_compaction_bytes"] =
-          std::to_string(cf_stats_count_[SOFT_PENDING_COMPACTION_BYTES_LIMIT]);
+      std::to_string(cf_stats_count_[SOFT_PENDING_COMPACTION_BYTES_LIMIT]);
   (*cf_stats)["io_stalls.memtable_compaction"] =
-          std::to_string(cf_stats_count_[MEMTABLE_COMPACTION]);
+      std::to_string(cf_stats_count_[MEMTABLE_COMPACTION]);
   (*cf_stats)["io_stalls.memtable_slowdown"] =
-          std::to_string(cf_stats_count_[MEMTABLE_SLOWDOWN]);
+      std::to_string(cf_stats_count_[MEMTABLE_SLOWDOWN]);
+
+  uint64_t total_stop =
+      cf_stats_count_[LEVEL0_NUM_FILES_TOTAL] +
+      cf_stats_count_[HARD_PENDING_COMPACTION_BYTES_LIMIT] +
+      cf_stats_count_[MEMTABLE_COMPACTION];
```

Comment:
we should rename these constants (unrelated to your diff), they aren't descriptive at all.